pkg/kyma/cmd/test/run/cmd.go

```diff
@@ -0,0 +1,155 @@
+package run
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"time"
+
+	oct "github.com/kyma-incubator/octopus/pkg/apis/testing/v1alpha1"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test"
+	"github.com/kyma-project/cli/pkg/kyma/cmd/test/client"
+	"github.com/kyma-project/cli/pkg/kyma/core"
+	"github.com/spf13/cobra"
+)
+
+type command struct {
+	opts *options
+	core.Command
+}
+
+func NewCmd(o *options) *cobra.Command {
+	cmd := command{
+		Command: core.Command{Options: o.Options},
+		opts:    o,
+	}
+
+	cobraCmd := &cobra.Command{
+		Use:     "run",
+		Short:   "Run tests on a running Kyma cluster",
+		Long:    `Run tests on a running Kyma cluster`,
+		RunE:    func(_ *cobra.Command, _ []string) error { return cmd.Run() },
+		Aliases: []string{"r"},
+	}
+
+	cobraCmd.Flags().StringVarP(&o.Name, "name", "n", "", "Name for the new test suite")
+	cobraCmd.Flags().StringVarP(&o.Tests, "tests", "s", "", "Test names to execute. Example: --tests=cluster-users-test,test-api-controller-acceptance. Overlaps with --all")
+	cobraCmd.Flags().BoolVarP(&o.RunAll, "all", "a", false, "Run all test cases in the test suite. Overlaps with --tests")
```

Comment:
Default should run all suites, please do not have an "all" flag

Reply:
Fixed. The`-all` flag remove completely.